cppcache/src/TypeRegistry.cpp

```diff
@@ -25,8 +25,9 @@
 
 TypeRegistry::TypeRegistry(CacheImpl* cache) : m_cache(cache) {}
 
-void TypeRegistry::registerType(TypeFactoryMethod creationFunction) {
-  m_cache->getSerializationRegistry()->addType(creationFunction);
+void TypeRegistry::registerType(TypeFactoryMethod creationFunction,
+                                uint32_t id) {
```

Comment:
As above, pretty sure this should be `int32_t` to match Java/Server API.

Reply:
Done.